src/views/v2/tickets/item.php

```diff
@@ -11,8 +11,9 @@
  * @link    https://evnt.is/1amp Help article for RSVP & Ticket template files.
  *
  * @since   5.0.3
+ * @since TBD Display data attributes via `tribe_attributes` and make them filterable via ``.
```

Comment:
```suggestion
 * @since TBD Display data attributes via `tribe_attributes` and make them filterable via `tribe_tickets_block_ticket_html_attributes `.
```